### PR TITLE
Add troubleshooting script

### PR DESCRIPTION
**Pull request by z3ntu**
I couldn't really test all branches of the script but theoretically it should work :wink:

### CODE REVIEW

File: scripts/troubleshoot.sh [hunk 1]
```diff
@@ -0,0 +1,89 @@
+#!/bin/bash
+
+dkms_name="razer_chroma_drivser"  # TODO Rename when dkms module is renamed.
```
Comment (terricain) [inline scripts/troubleshoot.sh]: drivser?

File: scripts/troubleshoot.sh [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+
+# Step 2 - check if a module is already loaded.
+lsmod=$(lsmod | grep razer)
+if [ -z "$lsmod" ]; then
+  echo "INFO: There is no kernel module loaded."
+fi
+
+# Step 3 - check if the module can load correctly
+echo "Please enter your 'sudo' password for priviliged operations."
+module_name="razerkbd"
```
Comment (terricain) [inline scripts/troubleshoot.sh:22]: Just require uid=0 from the start

File: scripts/troubleshoot.sh [hunk 3]
```diff
@@ -0,0 +1,89 @@
...
+    exit 1
+  fi
+fi
+
+# Step 4 - check the daemon
+log_dir=/tmp/razer_logs
+mkdir -p $log_dir
+razer-service -r -v --log-dir=$log_dir
+sleep 3
+#daemon_output=$(cat $log_dir/razer.log)  # TODO REVERT
```
Comment (terricain) [inline scripts/troubleshoot.sh]: ...

File: scripts/troubleshoot.sh [hunk 4]
```diff
@@ -0,0 +1,89 @@
...
+if [ ! -z "$(echo $daemon_output | grep 'PermissionError: [Errno 13] Permission denied')" ]; then
+  echo "ERROR: The current user doesn't have permission to access the driver files. Checking why."
+  file=$(echo $daemon_output | cut -d "'" -f4)
+  owning_group=$(stat -c '%G' $file)
+  if [ $owning_group != "plugdev" ]; then
+    echo "FATAL: Because of some reason, the owner of the file isn't plugdev, as it should be. Exiting."
+    exit 1
+  fi
+  group_plugdev=$(groups | grep plugdev)
+  if [ -z "$group_plugdev" ]; then
```
Comment (terricain) [inline scripts/troubleshoot.sh:73]: You could just check for plugdev group no need to check daemon output for that

File: scripts/troubleshoot.sh (hunk 2 above)
Comment (z3ntu) [reply scripts/troubleshoot.sh:22]: I actually don't want the whole script to be run as root so I think this should work just fine...

File: scripts/troubleshoot.sh (hunk 4 above)
Comment (z3ntu) [reply scripts/troubleshoot.sh:73]: If there are more common problems in the daemon log, I want to add them in that if too so yeah... Theoretically yes but it doesn't matter I think ;)

**Comment by z3ntu** [comment]
You didn't see my testing changes whooosh 😜

File: scripts/troubleshoot.sh (hunk 2 above)
Comment (terricain) [reply scripts/troubleshoot.sh:22]: You forget my young padawan, my word is law... lol np, as long as it works i aint fussed.

**Pull request merged**